### PR TITLE
[13.0][MIG] business_requirement_sale: Migration to 13.0

### PR DESCRIPTION
**Pull request by Tardo**
depends #333 

cc @tecnativa TT19719

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can you rebase?

**Comment by Tardo** [comment]
rebased

**Comment by Tardo** [comment]
Rebased again...

**Comment by pedrobaeza** [comment]
Now it's when you need to rebase :wink:

File: business_requirement_sale/views/business_requirement_views.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+            ref="business_requirement.view_business_requirement_form"
+        />
+        <field name="groups_id" eval="(4, ref('sales_team.group_sale_salesman'))" />
+        <field name="arch" type="xml">
+            <xpath expr='//div[@name="button_box"]' position='inside'>
+                <button
+                    class="oe_stat_button"
+                    type="object"
+                    name="open_orders"
+                    icon="fa-usd"
```
Comment (joao-p-marques) [inline business_requirement_sale/views/business_requirement_views.xml]: Seems like this icon has changed with [FA v5](https://fontawesome.com/icons/dollar-sign), which Odoo v13 uses. Can this be a problem?

**Comment by Tardo** [comment]
All green

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@joao-p-marques please recheck

**Review by joao-p-marques — APPROVED** [review]
LGTM :+1:

**Pull request merged**